site/_posts/2018-01-31-troubleshooting-auth.md

```diff
@@ -331,11 +331,13 @@ by ManageIQ authentication are correctly provided by the underlying technology.
   --print-reply \
   --system \
   --dest=org.freedesktop.sssd.infopipe /org/freedesktop/sssd/infopipe org.freedesktop.sssd.infopipe.GetUserAttr \
-  string:mshiffrin array:string:mail,givenname,sn,displayname
+  string:mshiffrin array:string:mail,givenname,sn,displayname,domainname
 </code></pre>
 
 The output from the above ***dbus-send / GetUserAttr*** command should look similar to the following:
 
+`Note:` The ***domainname*** attribute is only available in newer releases of ManageIQ.
```

Comment:
Can you specify what release? e.g. Gaprindashvili beta/GA/2 etc.

Reply:
@cybette Good idea. Done. Thank you. JoeV